src/main/resources/assets/iris/lang/en_us.json

```diff
@@ -0,0 +1,5 @@
+{
+  "iris.shaders.reloaded": "Reloaded Shaders!",
+  "iris.keybind.reload": "Reload Shaders",
+  "iris.keybinds": "Iris Keybinds"
```

Comment:
I think the convention is to not have a "Keybinds" part, so just an "Iris" section instead.